### PR TITLE
docs(nomos): Add doxygen comments for nomos

### PR DESCRIPTION
**Pull request by GMishx**
## Description

Added missing doxygen comments for nomos.

## How to test

Current documentation: https://gmishx.github.io/ (updated on every commit).

Please merge after #1098

### CODE REVIEW

File: fossology_doxygen.conf [hunk 1]
```diff
@@ -0,0 +1,2427 @@
+# Doxyfile 1.8.11
```
Comment (maxhbr) [inline fossology_doxygen.conf]: This file has no license header

File: src/nomos/agent_tests/Unit/test_nomos_gap.c [hunk 2]
```diff
@@ -23,67 +27,102 @@ with this program; if not, write to the Free Software Foundation, Inc.,
...
+ * -# Call collapseInvisible()
+ * -# Check if the string is collapsed with getPairPosOff()
+ */
+void test_collapseInvisible()
+{
   char* buf, *fer;
   GArray* po;
-  buf=g_strdup_printf("\377abc\377\377de\377\377fg\377hi");
-  fer=g_strdup(buf);
+  buf = g_strdup_printf("\377abc\377\377de\377\377fg\377hi");
```
Comment (maxhbr) [inline src/nomos/agent_tests/Unit/test_nomos_gap.c]: Unnecessary reformatting (Formatting should only be done when the line is actually modified).

File: src/nomos/agent_tests/Unit/test_nomos_gap.c [hunk 3]
```diff
@@ -1,20 +1,24 @@
 /*
-Copyright (C) 2014, Siemens AG
+ Copyright (C) 2014, Siemens AG
```
Comment (maxhbr) [inline src/nomos/agent_tests/Unit/test_nomos_gap.c]: Unnecessary reformatting

File: fossology_doxygen.conf (hunk 1 above)
Comment (GMishx) [reply fossology_doxygen.conf]: Added in #1098

**Pull request merged**